application/templates/cms_autosave/macros.html

```diff
@@ -269,20 +231,21 @@
 {%- endmacro %}
 
 
-{% macro editable_department_select(field, value, organisations_by_type) -%}
+{% macro editable_department_select(field, organisations_by_type, value_text=None, value_id=None) -%}
```

Comment:
Bit unclear what `value_text` and `value_id` mean in this context, at first glance. Perhaps `selected_organisation_name` and `selected_organisation_id` might be clearer?  (Also perhaps we should rename the macro as `editable_organisation_select` as strictly speaking they're not all departments, eg NHS Digital or Start Up Loans Company.)

Reply:
The existing CMS form uses `department_select`. which is where I got the name here from.  But we can fix it to org while we're here.